src/rpc/protocol.h

```diff
@@ -74,6 +74,8 @@ enum RPCErrorCode
     RPC_WALLET_WRONG_ENC_STATE      = -15, //! Command given in wrong wallet encryption state (encrypting an encrypted wallet etc.)
     RPC_WALLET_ENCRYPTION_FAILED    = -16, //! Failed to encrypt the wallet
     RPC_WALLET_ALREADY_UNLOCKED     = -17, //! Wallet is already unlocked
+    RPC_WALLET_INSYNC               = -18, //! Blockchain is in sync
+
```

Comment:
Remove this (it collides with an upstream error code, and in any case is redundant with `RPC_CLIENT_IN_INITIAL_DOWNLOAD`).